src/Traits/DetectsChanges.php

```diff
@@ -50,6 +58,20 @@ public function attributeValuesToBeLogged(string $processingEvent): array
 
     public static function logChanges(Model $model): array
     {
-        return collect($model)->only($model->attributesToBeLogged())->toArray();
+        return collect($model->attributesToBeLogged())->mapWithKeys(
+            function ($value) use ($model) {
+                if (is_iterable($value)) {
```

Comment:
`is_iterable` only exists in PHP 7.1, I'm not ready to drop PHP 7.0 support just yet.